console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx

```diff
@@ -50,59 +50,72 @@ const FinishedStep: React.FunctionComponent<IFinishedStepProps> = ({
   };
   return (
     <>
-      {!success ? (
-        <EmptyState
-          variant={EmptyStateVariant.full}
-          className={css(styles.empty_state)}
-        >
-          <EmptyStateIcon
-            icon={ExclamationCircleIcon}
-            className={css(styles.cog_red_color)}
-          />
-          <Title headingLevel="h5" size="xl">
-            The project cannot be created
-          </Title>
-          <br />
-          <EmptyStateBody>
-            Error occured during {projectType} Project creation for management,
-            return to homepage to view all projects
-          </EmptyStateBody>
-          <br />
-          <br />
-          <Button variant="link" onClick={onClose}>
-            Cancel
-          </Button>
-        </EmptyState>
-      ) : (
-        <EmptyState
-          variant={EmptyStateVariant.full}
-          className={css(styles.empty_state)}
-        >
-          <EmptyStateIcon
-            icon={CheckCircleIcon}
-            className={css(styles.cog_green_color)}
-          />
-          <Title headingLevel="h5" size="xl">
-            Creation successful
-          </Title>
-          <br />
-          <EmptyStateBody>
-            Enter your {projectType} Project for management, or return to
-            homepage to view all projects.
-          </EmptyStateBody>
-          <br />
-          <Link to={projectDetailUrl()}>
-            <Button variant={ButtonVariant.primary} component="a">
-              View the project
+      {success !== undefined &&
+        (!success ? (
+          <EmptyState
```

Comment:
can we wrap this code into two seperate functions success and failure cases.
functions: renderSuccessState and renderFailedState. i think it would more cleaner code. wdyt ?

Reply:
Agreed. Will make this change

Reply:
Made the changes as suggested